mappings/net/minecraft/block/entity/SignBlockEntity.mapping

```diff
@@ -24,3 +27,14 @@ CLASS net/minecraft/class_2625 net/minecraft/block/entity/SignBlockEntity
 		ARG 1 value
 	METHOD method_30843 getTextOnRow (IZ)Lnet/minecraft/class_2561;
 		ARG 1 row
+		ARG 2 filtered
+	METHOD method_33384 unparsedTextFromJson (Ljava/lang/String;)Lnet/minecraft/class_2561;
```

Comment:
```suggestion
	METHOD method_33384 unparseTextFromJson (Ljava/lang/String;)Lnet/minecraft/class_2561;
```

Reply:
This text returned is just like text from json, but the resulting score/nbt text etc aren't immediately resolved. Imo unparsed is better than unparse, and feel free to post more suggstions

Reply:
Make sense then 👍